spec/App/Entity/PrivateDiscussionSpec.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace spec\App\Entity;
+
+use App\Entity\PrivateDiscussion;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use App\Entity\User;
+
+class PrivateDiscussionSpec extends ObjectBehavior
+{
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(PrivateDiscussion::class);
+    }
+
+    function it_sets_the_admin(User $user)
+    {
+        $user->addAdminDiscussion($this)->willReturn(null);
```

Comment:
Useless. Unless you want to check it is called. Then you would use `$user->addAdminDiscussion($this)->shouldBeCalled()`.